services/QuillLMS/engines/evidence/lib/evidence/engine.rb

```diff
@@ -20,5 +22,9 @@ class Engine < ::Rails::Engine
 
       g.fallbacks[:shoulda] = :test_unit
     end
+
+    initializer "evidence.factories", after: "factory_bot.set_factory_paths" do
```

Comment:
Should this initializer block be wrapped in a Rails.env.test? conditional to prevent it loading in development and production.

Reply:
Actually, probably 

```
unless Rails.env.production?
  initializer "evidence.factories", after: "factory_bot.set_factory_paths" do
          FactoryBot.definition_file_paths << File.expand_path('../../../spec/factories/evidence', __FILE__) if defined?(FactoryBot)
  end
end
```
 
I feel like the defined?(FactoryBot) might be redundant given the require 'factory_bot_rails' at the top of the file.